rasa/core/channels/facebook.py

```diff
@@ -141,7 +143,21 @@ def send(self, recipient_id: Text, element: Any) -> None:
         # this is a bit hacky, but the client doesn't have a proper API to
         # send messages but instead expects the incoming sender to be present
         # which we don't have as it is stored in the input channel.
-        self.messenger_client.send(element.to_dict(), recipient_id, "RESPONSE")
+        
+        elementlenght = element.to_dict()
+       
+        if 'text' in elementlenght:
```

Comment:
You need to format the code using black, see https://github.com/RasaHQ/rasa#formatting.
It will update your code to use double quotes instead of single quotes, e.g. 
```
        if "text" in elementlenght:
```